Blog/views.py

```diff
@@ -50,6 +51,28 @@ def add_comment(request, pk):
     return index(request)
 
 
+@login_required
+def add_post_form(request):
+    do_add_url = reverse('post-add-do')
+    context = {'do_add_url': do_add_url}
+    return render(request, 'Blog/post_add.html', context)
+
+
+@login_required
+def add_post_do_add(request):
+    if request.method == "POST":
+        content = request.POST.get("content", "")
```

Comment:
You are not validating the user input at all. This opens your website to some nasty attacks (XSS, etc).

You could use a class based `CreateView` for this kind of task in combination with a `ModelForm`.